supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc

```diff
@@ -236,7 +236,7 @@ Do not use "Firewire" or "firewire". Although FireWire is a trademark of Apple C
 [discrete]
 [[futexes]]
 ==== image:images/yes.png[yes] futexes (noun)
-*Description*: "Futex" is an abbreviation of "fast user-space mutex". "Futexes" is the correct plural form.
+*Description*: _Futex_ is an abbreviation of _fast user-space mutex_. _Futexes_ is the correct plural form.
```

Comment:
Can you make the two first sentences of "fuxed" and "futexes" match? They are slightly different (with one using italics and one using quotes for what it stands for).

Reply:
:heavy_check_mark: